webpack.config.js

```diff
@@ -79,3 +79,10 @@ if ( 'production' === process.env.NODE_ENV ) {
 } else {
 	config.devtool = 'source-map';
 }
+
+if ( 'test' === process.env.NODE_ENV ) {
+	config.target = 'node';
+	config.externals = [ nodeExternals() ];
+}
+
+module.exports = config;
```

Comment:
Maybe a preferred style, but just FYI this would have still worked fine if left as it was.